components/case_macros/src/lib.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
You need a Cargo.toml for this new crate, see this example: https://github.com/tikv/tikv/pull/10185/files#diff-085a984383b78dc773ce73d7bcba1b36b00f305dc78dd3967fc22faa89a162fb

Reply:
I did not noice that `Cargo.toml` file has not been add to git. everything goes OK but failed in CI, Silly mistake